mx_gate.py

```diff
@@ -253,7 +254,7 @@ def gate(args):
                 if mx.command_function('pylint')(['--primary']) != 0:
                     _warn_or_abort('Pylint not configured correctly. Cannot execute Pylint task.', args.strict_mode)
 
-        gate_clean(cleanArgs, tasks, tags=[Tags.build, Tags.style])
+        gate_clean(cleanArgs, tasks, tags=[Tags.build, Tags.fullbuild])
```

Comment:
`fullbuild` should really imply `build` but that's probably adding unnecessary complexity to the tags system.